src/projectscene/view/clipsview/selectionviewcontroller.cpp

```diff
@@ -24,7 +24,7 @@ void SelectionViewController::onPressed(double x, double y)
 
     if (!selectionController()->isDataSelected()) {
         TrackIdList tracks = determinateTracks(m_startPoint.y(), y);
```

Comment:
This confused me for a while, I thought we might really pass a range, but in fact, looking at line 22 shows that `m_startPoint.y()` just equals `y` at this stage.
At least please replace `m_startPoint.y()` with just `y` to help the slow-minded of my kind understand your code better. You could even have a `determinateTrack(double y)` that'd internally just call `determinateTracks(y, y)`, but that'd be a stretch.